### PR TITLE
[New] Add tailwindcss@0.4.1 w/ npm auto-update

### PR DESCRIPTION
**Pull request by finch185277**
Pull request for issue: #12372

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/tailwindcss/tailwindcss
 * Official website (optional, not the repository):https://tailwindcss.com
 * NPM package url (optional):https://www.npmjs.com/package/tailwindcss
 * License and its reference:MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:186
   - Count of stars:3892
   - Count of forks:199
 * NPM download stats (optional):
   - Downloads in the last day:552
   - Downloads in the last week:3103
   - Downloads in the last month:11148

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by finch185277** [comment]
in `v0.2.1` `v0.1.5`,
no `preflight.css` `tailwind.css` `utilities.css` in dist folder.

File: ajax/libs/tailwindcss/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "keywords": [
+    "tailwindcss",
+    "css",
+    "css-framework",
+    "postcss",
+    "functional-css",
+    "utility-classes",
+    "responsive"
+  ],
+  "author": "Adam Wathan",
```
Comment (extend1994) [inline ajax/libs/tailwindcss/package.json]: Let's use contributors info from the upstream
https://github.com/tailwindcss/tailwindcss/blob/master/package.json#L13-L17

Comment (finch185277) [reply ajax/libs/tailwindcss/package.json]: I use  
```
 "authors": [
    "Adam Wathan <adam.wathan@gmail.com>",
    "Jonathan Reinink <jonathan@reinink.ca>",
    "David Hemphill <davidlee.hemphill@gmail.com>"
  ],
```

**Comment by finch185277** [comment]
Ok, I just leave these two versions.

**Pull request merged**